tests/testmusig2.c

```diff
@@ -71,11 +72,18 @@ MUSIG2_ERROR musig2_helper_verify(unsigned char *serialized_pubkey_list, unsigne
     return MUSIG2_OK;
 }
 
+void musig2_helper_fuzz_data(unsigned char *serialized_pubkey, int size){
+    int i;
+    for (i = 0; i <  size; i++)
+        serialized_pubkey[i] = rand();
+}
+
 #include "functiontest.c"
 #include "serdetest.c"
 }
 
 int main(int argc, char* argv[]){
+    srand ((unsigned int) time (NULL));
```

Comment:
What is this for?

Reply:
It seeds the RNG (`rand()`) that we use to set a random index to fuzz one of the public keys in the list.

Reply:
Ok, sounds good. Looks good to me then!